src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -114,7 +114,10 @@ static ExperimentalFeature()
                     description: "New parameter set for ForEach-Object to run script blocks in parallel"),
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-linux`
assignment statement parsing.Parse error expected: <<(Get-Variable x) ??= 1>>.Error position (iteration:0)
```powershell
Expected 0, but got 1.
at <ScriptBlock>, /home/vsts/work/1/s/test/tools/Modules/HelpersLanguage/HelpersLanguage.psm1: line 105
105:             It "Error position (iteration:$i)" -Pending:$SkipAndCheckRuntimeError { $acutalPostion | Should Be $expectedOffsets[$i] }
```